contrib/natss/pkg/dispatcher/dispatcher/dispatcher.go

```diff
@@ -67,8 +64,17 @@ func createReceiverFunction(s *SubscriptionsSupervisor, logger *zap.SugaredLogge
 		logger.Infof("Received message from %q channel", channel.String())
 		// publish to Natss
 		ch := getSubject(channel)
+		if s.natssConn == nil {
+			return fmt.Errorf("No Connection to NATS")
+		}
 		if err := stanutil.Publish(s.natssConn, ch, &m.Payload, logger); err != nil {
 			logger.Errorf("Error during publish: %v", err)
+			if err.Error() == stan.ErrConnectionClosed.Error() {
+				logger.Error("Connection to NATS has been lost, attempting to reconnect.")
```

Comment:
NATS -> NATSS

Here and all the other comments and log lines.

Reply:
done